docs/src/pages/demos/buttons/buttons.md

```diff
@@ -67,6 +67,14 @@ Sometimes you might want to have icons for certain button to enhance the UX of t
 
 {{"demo": "pages/demos/buttons/IconLabelButtons.js"}}
 
+## Customized Buttons
+
+You have been reading our [overrides documentation page](/customization/overrides)
+but you are not confident jumping in?
+Here is an example of how you can change the main color of an input. There is no limit.
+
```

Comment:
"If you have been reading the [overrides documentation page](/customization/overrides)
but you are not confident jumping in, here's an example of how you can change the main color of a Button."

> There is no limit. 

To what exactly?